includes/controls/groups/typography.php

```diff
@@ -86,7 +86,8 @@ public static function get_type() {
 	protected function init_fields() {
 		$fields = [];
 
-		$default_fonts = Plugin::$instance->kits_manager->get_current_settings( 'default_generic_fonts' );
+		// TODO: IMPLEMEMENT SOLUTION IN FRONTEND
+		$default_fonts = '';// Plugin::$instance->kits_manager->get_current_settings( 'default_generic_fonts' );
```

Comment:
?

Reply:
This needs to be handled by Mati

Reply:
@matipojo ?

Reply:
Because of the bug that mixes controls from kit, it's temporarily disabled.
@shilo-ey should decide how to handle it. maybe we will remove the feature.